src/oneid/oneid-lambda-assertion/requirements.txt

```diff
@@ -0,0 +1,2 @@
+boto3==1.20.24
```

Comment:
I don't think this file is required. Boto3 is already included in the runtime. 
We already deployed the function without any additional packages so I'm wondering why they are now needed.